### PR TITLE
Add hello world Vue app and cdk initialized skeleton

### PR DESCRIPTION
**Pull request by justinebreuch**
I didn't add anything other than a TODO line about initializing the CloudFormation stack. This is the result of running `cdk init` and `vue create` to just get a hello world page up.

### CODE REVIEW

**Comment by beekley** [comment]
Before we merge this, can you test that everything builds and runs? I'm not sure the best way to do that, but it's probably running `npm run build` in `cdk` directory, and `npm run serve` in the client (then seeing if you can load the HTML in a browser).

**Comment by justinebreuch** [comment]
The Vue app runs at your localhost:8080 address and just looks like https://screenshot.googleplex.com/MTAvwGCJQycHyXN.png

**Review by beekley — APPROVED** [review]
(oops, I thought I already approved this)

Left a few small comments but LGTM as the baseline.

**Review by kailamjeter — APPROVED** [review]

File: open-data-platform.iml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
[the remark this answers is not in the document]
Comment (justinebreuch) [reply open-data-platform.iml:1]: Added some commentary to the files I knew based on suggestions in https://vue-styleguidist.github.io/docs/Documenting.html#code-comments

**Pull request merged**